.github/workflows/ci.yml

```
@@ -0,0 +1,39 @@
+name: CI
+
+on:
```

yamllint (1.35.1)

[warning] truthy value should be one of [false, true]

(truthy)